ecopod/sendRawTransactionConditional.md

```diff
@@ -0,0 +1,53 @@
+# Purpose
+
+To fully unlock ERC-4337 on the OP-Stack.
+
+# Summary
+
+By providing an auxilliary transaction submission mechanism, [eth_sendRawTransactionConditional](https://notes.ethereum.org/@yoav/SkaX2lS9j), we can enable Bundlers to submit Entrypoint transactions with stronger guarantees, avoiding costly inadvertent reverts.
```

Comment:
Can you describe the semantics of the rpc in a tldr fashion?

Reply:
added